### ISSUE TITLE
Add `+Err/L2_Lat` for confusing Cyrillic with Latin letters

### ISSUE THREAD

**Issue by reynoldsnlp** [giellalt/lang-rus#15]
Words like `вилет` (cf `билет`). The following are candidates to consider:

```
б > в
й > у
н > п
п > р
к > с
х > н
```

**Comment by Trondtr** [comment]
Yes, good idea. If you meen the proofing tool they should go in the editdist.default.txt (I made a suggestion). But perhaps you mean spellrelax, so that a tolerant analyser may recognise them? To me it seems this type of error will be more prominent in western writers' Russian. I myself write on яешрты, and often write гоборить and цовет, since I am supposed to press V and S to get в and c, but then instead press B and C.
